src/server/apis/entries-api.js

```diff
@@ -10,48 +10,19 @@ router.get('/', (req, res) => {
 });
 
 router.delete('/:id', (req, res) => {
```

Comment:
Yah, we're not actually using the params at all here, so we should just take that out